sql/Pg-database.sql

```diff
@@ -1167,7 +1167,6 @@ CREATE TABLE cr_report (
     approved_by int references entity(id),
     approved_username text,
     recon_fx bool default false,
-    max_ac_id int,
```

Comment:
max_ac_id makes all SL based migration of reconciliations fail because it assumes that no future submitted record exist. When we migrate, they all are there and that pulls all transactions in each cr_report.

Reply:
max_ac_id usage ensure failure for SL migrations